3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,87 @@
+//Age-ify (A future age calculator)
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log('you will be' + " " + age + ' yeras old in' + " " + yearFuture);
+
```

Comment:
👍 Great implementation of the age calculation.
🚩 Improvement: Use const instead of let if the variable isn't going to be reassigned later.
💡 Consider using template literals to make the string concatenation cleaner